portal/src/components/Collections/Collection/Collection.component.js

```diff
@@ -208,33 +165,22 @@ export default {
     }
   },
   watch: {
-    /**
-     * Watcher on the search field
-     * @param search - String
-     */
     search(search) {
       this.$emit('input', search)
     },
-    /**
-     * Watcher on the contentEditable field
-     * @param isEditable - Boolean
-     */
     contenteditable(isEditable) {
       if (!isEditable) {
         this.resetData()
       }
     },
-    /**
-     * Watcher on the collection object
-     * @param collection - Object
-     */
     collection(collection) {
-      if (!_.isEmpty(collection)) {
-        const title =
-          this.$i18n.locale === 'nl' ? collection.title_nl : collection.title_en
-        this.setCollectionTitle(title)
+      if (collection) {
```

Comment:
Hier idem. De isEmpty check is defensief programmeren. Ik heb hier te vaak mensen op vast zien zitten (inclusief mijzelf in het verleden)